RoeiVerenigingWPF/Pages/MainPage.xaml.cs

```diff
@@ -1,16 +1,39 @@
+using DataAccessLibrary;
+using RoeiVerenigingLibrary.Model;
+using RoeiVerenigingLibrary.Services;
 using RoeiVerenigingWPF.Frames;
+using RoeiVerenigingWPF.Pages.Member;
 using System.Windows.Controls;
+using System.Windows.Input;
 
 namespace RoeiVerenigingWPF.Pages
 {
     public partial class MainPage : Page
     {
+        //TODO: dit aanpassen naar database ophalen, skylabs ligt eruit
+        public List<Event> _events { get; set; }
+        private EventService _eventService = new(new EventRepository());
+
+
+        public Event eEvent { get; set; }
```

Comment:
klopt deze naam? properties moeten trouwens beginnen met een hoofdletter maar is hier kut

Reply:
event los als naam mag niet. ik heb dit opgelost door een s er achter te zetten. denk dat willeke er daarom een e voor heeft gezet

Reply:
maak er dan iets van "CurrentEvent" van of iets die richting in, is duidelijker dan eEvent of Events

Reply:
werd aangeraden door rider om te gebruiken